exec/java-exec/src/main/java/org/apache/drill/exec/work/fragment/FragmentExecutor.java

```diff
@@ -67,6 +88,11 @@
 
   private volatile RootExec root;
   private final AtomicReference<FragmentState> fragmentState = new AtomicReference<>(FragmentState.AWAITING_ALLOCATION);
+  /**
+   * Holds all of the messages sent by downstream recievers that have finished. The {@link FragmentExecutor#run()} thread reads from this queue and passes the
+   * finished messages to the fragment's {@link RootExec} via the {@link RootExec#receivingFragmentFinished(FragmentHandle)} method.
+   */
+  private final Queue<FragmentHandle> recieverFinishedQueue = new ConcurrentLinkedQueue<>();
```

Comment:
please fix typo (receiver)

Reply:
Thanks for catching. Fixed all the misspellings of receiver.